frontend/src/components/Recipe.js

```diff
@@ -3,7 +3,7 @@ import {Card, CardContent, Typography} from "@material-ui/core";
 
 export default function Recipe({recipe}) {
     return (
-        <ListItem key={recipe.id}>
+        <ListItem >
```

Comment:
Your app will be mobile first, right? To reflect all screen sizes you could switch to relative measures `margin: 0.2rem`

Reply:
thx for the headup, still figuring the frontend stuff out myself and haven't really given thought to any CSS shenanigans. But I'll keep this in mind when I get to that part.

Reply:
"shenanigans" 👍🏽